### ISSUE TITLE
Update to work with socket.io v1.4.4+

### ISSUE THREAD

**Issue by localyost3000** [swimmadude66/YTRadio#43]
introduced new breaking changes, needs to be accounted for

**Comment by acoulon99** [comment]
Did a bit of digging.
socket adapter(server) v1.3.4:
```
  adapter: 
   Adapter {
     nsp: 
      Namespace {
        name: '/chat',
        server: [Object],
        sockets: [Object],
        connected: [Object],
        fns: [],
        ids: 0,
        acks: {},
        adapter: [Circular],
        _events: [Object],
        _eventsCount: 1 },
     rooms: { xJMp1niKyFJvAWHcAAAA: [Object] },
     sids: { xJMp1niKyFJvAWHcAAAA: [Object] },
     encoder: Encoder {} },
  id: 'xJMp1niKyFJvAWHcAAAA',
```

socket adapter v1.4.4:
```
  adapter: 
   Adapter {
     nsp: 
      Namespace {
        name: '/chat',
        server: [Object],
        sockets: [Object],
        connected: [Object],
        fns: [],
        ids: 0,
        adapter: [Circular],
        _events: [Object],
        _eventsCount: 1 },
     rooms: { '/chat#EAVreIumDN8d05sYAAAA': [Object] },
     sids: { '/chat#EAVreIumDN8d05sYAAAA': [Object] },
     encoder: Encoder {} },
  id: '/chat#EAVreIumDN8d05sYAAAA',
```

**Comment by acoulon99** [comment]
It looks as if the server is giving a socket a unique ID per namespace in v1.4.4. One solution would be to have a single namespace(default `/`), then a separate room for `chat` and `media`.

**Comment by localyost3000** [comment]
Fixed in v0.5.0!

Seems that the angularJs socket.io library was just shite at handling namespaces. In the move to ng2+, I was able to use socket.io's official client, and it all just worked magically. Aint that a peach.
